packages/app/src/app/components/Overlay/Overlay.tsx

```diff
@@ -44,14 +44,23 @@ export const Overlay: React.FC<IOverlayProps> = ({
     height: 0,
   });
 
-  useLayoutEffect(() => {
+  const calculateBounds = () => {
     const posData = element.current.getBoundingClientRect();
     bounds.current = {
       top: posData.top,
       left: posData.left,
       width: posData.width,
       height: posData.height,
     };
+  };
+
+  useLayoutEffect(() => {
+    calculateBounds();
+
+    window.addEventListener('resize', calculateBounds);
+    return () => {
+      window.removeEventListener('resize', calculateBounds);
+    };
```

Comment:
I'm not very happy with this change, since the return value is used by React. If someone would change this function to something else that returns a function, then the code would start to behave differently. I like the explicitness of saying "don't do anything with the return value" by placing it in curly braces.